server/models/roundAnalysis.js

```diff
@@ -113,6 +113,12 @@ const schema = mongoose.Schema({
     minlength: 0,
     default: '',
   },
+  applicationCodes: {
```

Comment:
`roundIdList` has to be removed

Reply:
We need roundIdList to fetch courseEmployee Data. Without the roundId we cannot fetch data from UG. UG rest api is not updated yet. I asked Johannes regarding this. He said it will take two months to update UG rest api groups with the application codes. After that we can remove roundIds from here.

Reply:
Done